packages/components/src/globals/scss/_tooltip.scss

```diff
@@ -85,10 +101,8 @@
   // content box
   // @todo Simplify CSS selectors on next major release
   &::after,
-  &:hover .#{$prefix}--assistive-text,
-  &:focus .#{$prefix}--assistive-text,
-  &:hover + .#{$prefix}--assistive-text,
-  &:focus + .#{$prefix}--assistive-text {
+  .#{$prefix}--assistive-text,
+  + .#{$prefix}--assistive-text {
```

Comment:
Can you help me understand what this sibling selector is doing?

Reply:
prior to #3115 and #3151 we used `::after` pseudoelements to create the tooltip body. this pattern was changed in those PRs so that the tooltip body is now a div (`div.bx--assistive-text`), which can be a sibling or child of the tooltip trigger

the current PR just does these things: deduplicate and simplify selectors, and extract the styles for displaying the tooltip into a mixin

Reply:
Nice, do we still need the &::after rule then?

Reply:
yes until the next major version release (for backwards compatibility)